k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api
+spec:
+  releaseName: ccpay-refunds-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccpay-refunds-api
+    version: 0.1.4
+  values:
```

Comment:
Values section is incorrect.

Refer: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml#L11-L15

The image used in this config is correct.

Reply:
changes done